examples/config/gmm.yaml

```diff
@@ -41,15 +44,15 @@ training:
 
 evaluation:
   n_plots: 20 # number of times we visualise the model throughout training.
-  n_eval: 20 # for calculating metrics of flow w.r.t target.
+  n_eval: 20 # for calculating metrics of flow w.r.t target.list(np.linspace(0, n_iterations - 1, n_checkpoints, dtype="int"))
   eval_batch_size: 512 # must be a multiple of inner batch size
   n_checkpoints: 10 # number of model checkpoints saved
-  save_path:  results/many_well32/
+  save_path:  /home/laurence/work/code/FAB-TORCH/results/gmm/seed${training.seed}/snf/
 
 
 logger:
-  list_logger: true
-#  pandas_logger:
-#    save_period: 100 # how often to save the pandas dataframe as a csv
+#  list_logger: true
+  pandas_logger:
```

Comment:
I use wandb for logging but I have also made this pandas logger which just writes everything to a dataframe and saves that - is that okay for you?